docs/.vuepress/theme/components/AlgoliaSearch.vue

```diff
@@ -14,9 +14,20 @@ export default {
   },
   methods: {
     initialize(userOptions, lang) {
+
+      if (this.$page.currentVersion === 'next') {
+        // Disable search for next version as it's not crawled by Algolia
+        return;
+      }
+
+      const latestReleasedVersion = this.$page.versions.find(version => !isNaN(parseInt(version, 10)));
+      const isLatestReleasedVersion = latestReleasedVersion === this.$page.currentVersion;
```

Comment:
I think this can be simplified
```suggestion
      const isLatestReleasedVersion = this.$page.currentVersion === this.$page.latestVersion
```